src/plugins/data/public/search/aggs/metrics/parent_pipeline.test.ts

```diff
@@ -76,9 +74,7 @@ describe('parent pipeline aggs', function () {
         const field = {
           name: 'field',
           format: {
-            type: {
-              id: 'bytes',
-            },
+            toJSON: () => ({ id: 'bytes' }),
```

Comment:
These mocks were changed because this is basically the only part of field formats that AggConfig cares about: getting the serialized format for the field from the provided index pattern